src/components/router/FButtonLink.vue

```diff
@@ -24,16 +24,12 @@ template(v-else)
     slot
 </template>
 
-<style lang="stylus"></style>
-
 <script setup lang="ts">
-import { RouterLink } from 'vue-router';
-import { computed } from 'vue';
-
-import FButton from '@/components/FButton.vue';
-import { useSmartLink } from '@/composables/useSmartLink';
-
-import type { RouteLocationRaw, NavigationFailure } from 'vue-router';
+import type {
+  RouterLink,
+  RouteLocationRaw,
+  NavigationFailure,
+} from 'vue-router';
```

Comment:
Same point
```suggestion
import type {
  RouteLocationRaw,
  NavigationFailure,
} from 'vue-router';
import { RouterLink } from 'vue-router';
```

Reply:
Router components are globally imported, we only want to import types here :) 
But while checking, I discovered an issue with router link stories, thanks ! And I refactored stories related typings, if you want to have a check ! 

Thank you :)

Reply:
I agree with you, but I don't understand why we do import `RouterLink` then if it's globally imported ! Since RouterLink isn't a type ^^

We should only 
`import type {
  RouteLocationRaw,
  NavigationFailure,
} from 'vue-router';`

Reply:
Oh indeed, you're right! I thought it was used as type. Thanks 👌